lib/athena/Cargo.toml

```diff
@@ -3,7 +3,6 @@ name = "athena"
 authors.workspace = true
 edition.workspace = true
 version.workspace = true
-license = "MIT"
```

Comment:
The [`license`] field accepts the SPDX license expression (at least in case of crates.io), so maybe you want to change it to `MIT OR Apache-2.0` instead (the same goes for other crates).

[`license`]: <https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields>

Reply:
There we go. Every crate in the tree now has the correct SPDX ID and the respective licenses symlinked.